<?xml version="1.0"?>

<launch>
    <rosparam param="/vive_calibrating_node/controller_frame">"controller_LHR_FDBBBBC0"</rosparam>
    <rosparam param="/vive_calibrating_node/base_frame">"floor_base"</rosparam>
    <rosparam param="/vive_calibrating_node/tool_frame">"floor_tool0"</rosparam>
    <rosparam param="/vive_calibrating_node/test_frame">"controller_test"</rosparam>

    <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="static_tracker1_publisher" args="-0.124 0.046 0.025 0.0 -1.57079632679 0.0 camera_house tracker1_desired" />
    <node pkg="tf2_ros" type="static_transform_publisher" name="static_tracker2_publisher" args="0.124 0.046 0.025 0.0 1.57079632679 0.0 camera_house tracker2_desired" /> -->

    <node name="parkmartin_node" pkg="vive_calibrating" type="parkmartin_node" output="screen" required="true"/>
    <node name="vive_robot_calibrating_node" pkg="vive_calibrating" type="vive_robot_calibrating_node" output="screen" required="true"/>
</launch>